spec/boxr/webhooks_spec.rb

```diff
@@ -0,0 +1,30 @@
+# rake spec SPEC_OPTS="-e \"invokes webhook operations"\"
+describe "webhook operations" do
+  it 'invokes webhook operations' do
+    puts 'create webhook'
+    resource_id = @test_folder.id
+    type = 'folder'
+    triggers = ['FOLDER.RENAMED']
+    address = 'https://example.com'
+    new_webhook = BOX_CLIENT.create_webhook(resource_id, type, triggers, address)
+    new_webhook_id = new_webhook.id
+    expect(new_webhook.created_at).to_not be_empty
```

Comment:
I think this spec would be a bit more useful if it verified one of the data points - 

```suggestion
    expect(new_webhook.triggers.to_a).to eq(triggers)
```